accounts/account_manager.go

```diff
@@ -208,3 +219,35 @@ func zeroKey(k *ecdsa.PrivateKey) {
 		b[i] = 0
 	}
 }
+
+func (am *Manager) Export(path string, addr []byte, keyAuth string) error {
+	key, err := am.keyStore.GetKey(addr, keyAuth)
+	if err != nil {
+		return err
+	}
+	return crypto.SaveECDSA(path, key.PrivateKey)
```

Comment:
whenever theres a load, theres a save. know this and never be sorry.
but writing with proper permissions is great idea.